### PR TITLE
Bugfix/reset index drop

### LINKED ISSUE #25: Drop index in statcast postprocessing
Hi @jldbc - thanks for putting this project together. I am using the `statcast` function to retrieve pitching data, and am proposing a simple change.

Adding `drop=True` in the `reset_index()` call on [line 170 of `statcast.py`](https://github.com/jldbc/pybaseball/blob/master/pybaseball/statcast.py#L170) prevents an unnecessary column named "index". Happy to add change in a future PR.

Thanks again!

### PR DESCRIPTION
**Pull request by TheCleric**
Fixes the bug mentioned in #25. Also fixed up some formatting and long lines found via pylint and autopep8.
(closes #25)

### CODE REVIEW

File: pybaseball/playerid_lookup.py [hunk 1]
```diff
@@ -3,12 +3,13 @@
 import io
 import os
 
-# dropped key_uuid. looks like a has we wouldn't need for anything.
+# Dropped key_uuid. Looks like a has we wouldn't need it for anything.
```
Comment (schorrm) [inline pybaseball/playerid_lookup.py:6]: Way to fix the second mistake here, though I don't know what the first was supposed to be to begin with. I have no clue at all what this comment means, tbh

File: pybaseball/playerid_lookup.py [hunk 2]
```diff
@@ -18,43 +19,51 @@ def chadwick_register(save: bool = False) -> pd.DataFrame:
 
     print('Gathering player lookup table. This may take a moment.')
     s = requests.get(url).content
-    mlb_only_cols = ['key_retro', 'key_bbref', 'key_fangraphs', 'mlb_played_first', 'mlb_played_last']
+    mlb_only_cols = ['key_retro', 'key_bbref',
```
Comment (schorrm) [inline pybaseball/playerid_lookup.py:22]: I don't like this. This was col 103 and splitting this doesn't really harm readability

File: pybaseball/playerid_lookup.py [hunk 3]
```diff
@@ -18,43 +19,51 @@ def chadwick_register(save: bool = False) -> pd.DataFrame:
...
+                     'key_fangraphs', 'mlb_played_first', 'mlb_played_last']
     cols_to_keep = ['name_last', 'name_first', 'key_mlbam'] + mlb_only_cols
     table = pd.read_csv(io.StringIO(s.decode('utf-8')), usecols=cols_to_keep)
 
-    table.dropna(how='all', subset=mlb_only_cols, inplace=True) # Keep only the major league rows
+    # Keep only the major league rows
+    table.dropna(how='all', subset=mlb_only_cols, inplace=True)
     table.reset_index(inplace=True, drop=True)
 
-    table[['key_mlbam', 'key_fangraphs']] = table[['key_mlbam', 'key_fangraphs']].fillna(-1)
```
Comment (schorrm) [inline pybaseball/playerid_lookup.py:28]: Cols 93 and 94 for these. Don't like

File: pybaseball/playerid_lookup.py [hunk 4]
```diff
@@ -66,17 +75,20 @@ def playerid_lookup(last=None, first=None, player_list=None):
...
         return results
 
     if first is None:
         results = table.loc[table['name_last'] == last]
     else:
-        results = table.loc[(table['name_last'] == last) & (table['name_first'] == first)]
-    #results[['key_mlbam', 'key_fangraphs', 'mlb_played_first', 'mlb_played_last']] = results[['key_mlbam', 'key_fangraphs', 'mlb_played_first', 'mlb_played_last']].astype(int) # originally returned as floats which is wrong
-    results = results.reset_index().drop('index', 1)
-    return results
+        results = table.loc[(table['name_last'] == last) &
```
Comment (schorrm) [inline pybaseball/playerid_lookup.py:88]: Again

**Comment by schorrm** [comment]
I'm really not happy with the autopep8 results, the substantive content however I do really want to merge.
Can you submit this as only the content changes, and then we can treat the formatting as a separate change? Perhaps we can discuss an automatic standard for formatting, and with consensus on something like that move forward?

File: pybaseball/playerid_lookup.py (hunk 2 above)
Comment (TheCleric) [reply pybaseball/playerid_lookup.py:22]: Yeah I'm not sure why it's splitting early there.

**Pull request closed without merging**